lib/vagrant-openshift/command/test_origin_image.rb

```diff
@@ -117,14 +118,14 @@ def execute
 cd $temp_dir
 
 # clone the image repo
-git clone #{source}
-cd #{source_dir}
+git clone "#{source}"
```

Comment:
No, this is one of the image repos:

https://github.com/openshift/vagrant-openshift/pull/413/files#diff-b5524b94a3165b89ca0779574f0c98c9R78

Typical values will be 

- https://github.com/openshift/mongodb
- https://github.com/openshift/mysql
- etc.

And FWIW `ref` is the pull request's number.